### PR TITLE
[WIP] Python 2 compat removal

### PR DESCRIPTION
**Pull request by auvipy**
clean-ups for celery 5. do not merge.

### CODE REVIEW

**Comment by auvipy** [comment]
let's not duplicate the effort. there are many docs/bug issues which need triage or patch.

File: celery/__init__.py [hunk 1]
```diff
@@ -1,12 +1,11 @@
-# -*- coding: utf-8 -*-
```
Comment (thedrow) [inline celery/__init__.py]: why did you remove this? It's not related to Python 2

Comment (auvipy) [reply celery/__init__.py]: # -*- coding: utf-8 -*- ? this? might be mistakenly.

Comment (thedrow) [reply celery/__init__.py]: No it isn't.
https://www.python.org/dev/peps/pep-0263/

Comment (auvipy) [reply celery/__init__.py]: https://github.com/pandas-dev/pandas/pull/26197 and django removed them as well

Comment (jdufresne) [reply celery/__init__.py]: Starting with Python 3.0, utf-8 is the default encoding of source files:

https://www.python.org/dev/peps/pep-3120/

This makes the utf-8 encoding cookie redundant.

**Comment by auvipy** [comment]
continued in https://github.com/celery/celery/pull/5684
